.github/workflows/main.yml

```diff
@@ -71,6 +71,12 @@ jobs:
         name: collection
         path: .cache/collection-tarballs
 
+    - name: Create ansible.cfg file
+      run: |
+       echo "[galaxy]" > ansible.cfg
+       echo "server = https://old-galaxy.ansible.com/" >> ansible.cfg
```

Comment:
Interesting... so the fix is to point to the old galaxy.  Do we have any idea how long this old instance will be maintained?